routes/api/user/update-user.js

```diff
@@ -7,5 +7,50 @@ var router = express.Router();
 
 console.log('update-user');
 
+router.get("/api/users/:id", function(req, res) {
+    // Add sequelize code to find a single post where the id is equal to req.params.id,
+    db.Users.update({
+      user_name: req.body.user_name,
+      email: req.body.email,
+      about_me: req.body.about_me,
+      homepage_url: req.body.homepage_url,
+      linkedin_url: req.body.linkedin_url,
+      twitter_url: req.body.twitter_url,
```

Comment:
Let's add `zipcode: req.body.zipcode,` here to allow our new zipcode column to be updated